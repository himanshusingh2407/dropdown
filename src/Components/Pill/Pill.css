.pill {
    background-color: #eeeeee;
    padding: 4px 14px;
    border-radius: 25px;
    height: 24px;
    display: flex;
    align-items: center;
    margin-right: 5px;
    font-size: 14px;
}

.pill .content {
    max-width: 100px;
    overflow: hidden;
    text-overflow: ellipsis;
    white-space: nowrap;
}

.delete-pill {
    padding: 0 4px;
    cursor: pointer;
    font-weight: bold;
    font-size: 16px;
}