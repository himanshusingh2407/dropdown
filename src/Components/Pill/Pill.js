import React from 'react';
import './Pill.css'; 

function Pill({text, onDelete}) {
    return (
        <div className="pill">
            <span className="content">{text}</span>
            <span onClick={onDelete} className="delete-pill">x</span>
        </div>
    )
}

export default Pill;