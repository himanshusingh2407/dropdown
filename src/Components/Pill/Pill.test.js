import React from 'react';
import Pill from './Pill';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Pill text="Movie Name" onDelete={()=>{}}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});