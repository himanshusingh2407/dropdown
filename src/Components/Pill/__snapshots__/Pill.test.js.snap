// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="pill"
>
  <span
    className="content"
  >
    Movie Name
  </span>
  <span
    className="delete-pill"
    onClick={[Function]}
  >
    x
  </span>
</div>
`;
